ods_ci/tests/Tests/400__ods_dashboard/416__ods_dashboard_settings_accelerator_profiles.robot

```diff
@@ -0,0 +1,153 @@
+    Accelerator Profile Should Be Displayed In The Grid   ${ACC_DISPLAY_NAME}1
+    Verify Accelerator Profile Values Via CLI   ${ACC_DISPLAY_NAME}1   ${ACC_IDENTIFIER}
+    ...                                         ${ACC_DESCRIPTION}   ${ACC_ENABLED}    tolerations=yes
+    ...                                         tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                                         tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                                         tol_key=${ACC_TOLERATION_KEY}
+    ...                                         tol_value=${ACC_TOLERATION_VALUE}
+    ...                                         tol_seconds=${ACC_TOLERATION_SECONDS}
+
+Modify An Accelerator Profile Using "Accelerator Profiles" Administration UI
+    [Documentation]    Modify an Accelerator Profile instance from the Administration UI and verify it's content
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}2
+    Open ODS Dashboard With Admin User
+    Navigate To Page    Settings    Accelerator profiles
+    Edit Accelerator Profile    original_display_name=${ACC_DISPLAY_NAME}2
+    ...                         display_name=${ACC_DISPLAY_NAME}2_modified
+    ...                         identifier=${ACC_IDENTIFIER}_modified
+    ...                         description=${ACC_DESCRIPTION}_modified    tolerations=yes
+    ...                         tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                         tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                         tol_key=${ACC_TOLERATION_KEY}
+    ...                         tol_value=${ACC_TOLERATION_VALUE}
+    ...                         tol_seconds=${ACC_TOLERATION_SECONDS}
+    Accelerator Profile Should Be Displayed In The Grid   ${ACC_DISPLAY_NAME}2_modified
+    Verify Accelerator Profile Values Via CLI   ${ACC_DISPLAY_NAME}2_modified   ${ACC_IDENTIFIER}_modified
+    ...                                         ${ACC_DESCRIPTION}_modified   tolerations=yes
+    ...                                         tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                                         tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                                         tol_key=${ACC_TOLERATION_KEY}
+    ...                                         tol_value=${ACC_TOLERATION_VALUE}
+    ...                                         tol_seconds=${ACC_TOLERATION_SECONDS}
+
+Delete Tolerations from an Accelerator Profile Using "Accelerator Profiles" Administration UI
+    [Documentation]    Delete Tolerations from  an Accelerator Profile instance from the Administration UI and
+    ...                verify it's content
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}3
+    Open ODS Dashboard With Admin User
+    Navigate To Page    Settings    Accelerator profiles
```

Comment:
these 2 lines are repated in every test. Maybe could become a test setup

Reply:
Good point.
Done in 
1e6d6e0